src/theme/custom/cosmoDark/grid.js

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+// Set here the theme form ag grid
+import 'ag-grid-community/dist/styles/ag-theme-balham-dark.css';
+import 'ag-grid-community/dist/styles/ag-grid.css';
```

Comment:
`ag-grid-community/dist/styles/ag-grid.css` is still imported in the Table component, one of these imports should be removed, either here or in the ui Table component